sys/amd64/conf/GENERIC-MMCCAM

```diff
@@ -1,34 +1,24 @@
-# MMCCAM is the kernel config for doing MMC on CAM development
-# and testing on bhyve
+#
+# GEMERIC-MMCCAM
```

Comment:
s/GEMERIC/GENERIC/

Reply:
should fix it here https://github.com/freebsd/freebsd-src/blob/5d0826017fc9b84f50ef19b0d452bc0590072f2d/sys/arm/conf/GENERIC-MMCCAM#L2 as well then

Reply:
done.